web/templates/web/study-detail.html

```diff
@@ -148,7 +154,7 @@ <h4>Would you like to participate in this study?</h4>
                                 <option onemptied="" value="{{child.uuid}}|{{child.birthday}}">{{child.given_name}}</option>
                                 {% endfor %}
                             </select>
-                            <p class="text-warning" style="display:none" id='too-young'>Your child is still younger than the recommended age range for this study. If you can wait until he or she is old enough, we'll be able to use the collected data in our research! </p>
+                            <p class="text-warning" style="display:none" id='too-young'>Your child is still younger than the recommended age range for this study. If you can wait until <span id='wait-until'></span> he or she is old enough, we'll be able to use the collected data in our research! </p>
```

Comment:
I would remove this "until" here, so you don't have " f you can wait until X days until ...." days

Reply:
Thanks for catching that! "Until" now just in the span as a default so the sentence still makes sense if that text isn't replaced.